prisma/schema.prisma

```diff
@@ -156,6 +156,7 @@ model User {
   modules                   Module[]
   exercises                 Exercise[]
 
+  Exercise Exercise[] @relation("flaggedBy")
```

Comment:
Something like `flaggedExercises` will be a more suitable and descriptive name for this field.